acdtools

```diff
@@ -241,22 +241,26 @@ function ACDToolsUpload {
     fi
 
     # Upload to Amazon Drive
-    until ${ACDCLI} upload -o ${ACDEXCLUDE} ${MOUNTBASE}/local-encrypted/* \
-        ${ACDSUBDIR}
-    do
-        ULATTEMPTS=$((ULATTEMPTS+1))
-        ACDToolsLog error "Some uploads failed - uploading again after a sync \
-            (attempt ${ULATTEMPTS})"
-
-        if [ "${ULATTEMPTS}" -ge 3 ]; then
-            ACDToolsLog error "Uploads failed 3 (or more) times - Forcing full \
-                sync"
-            ACDToolsSyncNodes full
-        else
-            ACDToolsSyncNodes
-        fi
-        sleep 60
-    done
+	if [ "$(ls -A ${MOUNTBASE}/local-encrypted/*)" ]; then
+	    until ${ACDCLI} upload -o ${ACDEXCLUDE} ${MOUNTBASE}/local-encrypted/* \
+	        ${ACDSUBDIR}
+	    do
+	        ULATTEMPTS=$((ULATTEMPTS+1))
+	        ACDToolsLog error "Some uploads failed - uploading again after a sync \
+	            (attempt ${ULATTEMPTS})"
+
+	        if [ "${ULATTEMPTS}" -ge 3 ]; then
+	            ACDToolsLog error "Uploads failed 3 (or more) times - Forcing full \
+	                sync"
+	            ACDToolsSyncNodes full
+	        else
+	            ACDToolsSyncNodes
+	        fi
+	        sleep 60
+	    done
+	else
+		echo "${MOUNTBASE}/local-encrypted/* is empty - nothing to upload"
```

Comment:
Sure - I'm choosing `ACDToolsLog info` - let me know if that isn't as you'd like.